commercetools-models/src/main/java/io/sphere/sdk/shippingmethods/PriceFunction.java

```diff
@@ -0,0 +1,18 @@
+package io.sphere.sdk.shippingmethods;
+
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.sphere.sdk.annotations.ResourceValue;
+
+@ResourceValue
+@JsonDeserialize(as = PriceFunctionImpl.class)
+public interface PriceFunction {
+
+    String getCurrencyCode();
+
+    String getFunction();
+
+    static PriceFunction of(final String currencyCode, final String function) {
+        return new PriceFunctionImpl(currencyCode,function);
```

Comment:
I'm fine with this factory method, but we could generate a builder class here too with our @HasBuilder annotation.

Reply:
👍  good catch, will be included with therevisedd code.